api reader broken in 1.3.1

It looks like the simple_api_reader is broken in the latest release.

```
Error: Cannot find module '../elasticsearch_reader/elasticsearch_date_range/reader’
```

--- a/asset/asset.json
+++ b/asset/asset.json
@@ -1,4 +1,4 @@
 {
     "name": "elasticsearch",
-    "version": "1.3.2"
+    "version": "1.3.3"
 }

--- a/asset/package.json
+++ b/asset/package.json
@@ -1,6 +1,6 @@
 {
     "name": "asset",
-    "version": "1.3.2",
+    "version": "1.3.3",
     "main": "utils.js",
     "devDependencies": {},
     "scripts": {

--- a/asset/simple_api_reader/index.js
+++ b/asset/simple_api_reader/index.js
@@ -4,6 +4,7 @@ const { getOpConfig } = require('@terascope/job-components');
 const Promise = require('bluebird');
 const _ = require('lodash');
 const got = require('got');
+const reader = require('../elasticsearch_reader');
 const readerFn = require('../elasticsearch_reader/reader');
 const slicerFn = require('../elasticsearch_reader/elasticsearch_date_range/slicer');
 
@@ -218,7 +219,7 @@ function newReader(context, opConfig, jobConfig) {
 }
 
 function schema() {
-    const esSchema = require('../elasticsearch_reader').schema();
+    const esSchema = reader.schema();
     const apiSchema = {
         endpoint: {
             doc: 'The base API endpoint to read from: i.e. http://yourdomain.com/api/v1',

--- a/asset/simple_api_reader/index.js
+++ b/asset/simple_api_reader/index.js
@@ -4,6 +4,8 @@ const { getOpConfig } = require('@terascope/job-components');
 const Promise = require('bluebird');
 const _ = require('lodash');
 const got = require('got');
+const readerFn = require('../elasticsearch_reader/reader');
+const slicerFn = require('../elasticsearch_reader/elasticsearch_date_range/slicer');
 
 // eslint-disable-next-line
 process.env.NODE_TLS_REJECT_UNAUTHORIZED = '0';
@@ -207,12 +209,12 @@ function createClient(context, opConfig) {
 function newSlicer(context, executionContext, retryData, logger) {
     const opConfig = getOpConfig(executionContext.config, MODULE_NAME);
     const client = createClient(context, opConfig);
-    return require('../elasticsearch_reader/elasticsearch_date_range/slicer')(context, opConfig, executionContext, retryData, logger, client);
+    return slicerFn(context, opConfig, executionContext, retryData, logger, client);
 }
 
 function newReader(context, opConfig, jobConfig) {
     const client = createClient(context, opConfig);
-    return require('../elasticsearch_reader/elasticsearch_date_range/reader')(context, opConfig, jobConfig, client);
+    return readerFn(context, opConfig, jobConfig, client);
 }
 
 function schema() {

--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
     "name": "elasticsearch-assets",
     "description": "bundle of processors for teraslice",
-    "version": "1.3.2",
+    "version": "1.3.3",
     "main": "index.js",
     "scripts": {
         "lint": "eslint asset/**/*.js test/**/*.js",

